proposal/color-4-new-spaces-js.d.ts.md

```diff
@@ -48,67 +51,64 @@ import {Value} from '../spec/js-api/value';
 ### Color Space Definitions
 
 ```ts
-type ColorSpaceRGB =
-  | 'rgb'
-  | 'srgb'
-  | 'srgb-linear'
-  | 'display-p3'
-  | 'a98-rgb'
-  | 'prophoto-rgb';
 
-type ChannelNameRGB = 'red' | 'green' | 'blue';
+export type ColorSpaceHSL = 'hsl';
 
-type ColorSpaceHWB = 'hwb';
+export type ChannelNameHSL = 'hue' | 'saturation' | 'lightness';
 
-type ChannelNameHWB = 'hue' | 'whiteness' | 'blackness';
+export type ColorSpaceHWB = 'hwb';
 
-type ColorSpaceHSL = 'hsl';
+export type ChannelNameHWB = 'hue' | 'whiteness' | 'blackness';
 
-type ChannelNameHSL = 'hue' | 'saturation' | 'lightness';
+export type ColorSpaceLAB = 'lab' | 'oklab';
 
-type ColorSpaceXYZ = 'xyz' | 'xyz-d50' | 'xyz-d65';
+export type ChannelNameLAB = 'lightness' | 'a' | 'b';
 
-type ChannelNameXYZ = 'x' | 'y' | 'z';
+export type ColorSpaceLCH = 'lch' | 'oklch';
 
-type ColorSpaceLCH = 'lch' | 'oklch';
+export type ChannelNameLCH = 'lightness' | 'chroma' | 'hue';
 
-type ChannelNameLCH = 'lightness' | 'chroma' | 'hue';
+export type ColorSpaceRGB =
+  | 'a98-rgb'
+  | 'display-p3'
+  | 'prophoto-rgb'
+  | 'rgb'
+  | 'srgb'
+  | 'srgb-linear';
 
-type ColorSpaceLAB = 'lab' | 'oklab';
+export type ChannelNameRGB = 'red' | 'green' | 'blue';
 
-type ChannelNameLAB = 'lightness' | 'a' | 'b';
+export type ColorSpaceXYZ = 'xyz' | 'xyz-d50' | 'xyz-d65';
 
-type ChannelName =
-  | ChannelNameRGB
-  | ChannelNameHWB
+export type ChannelNameXYZ = 'x' | 'y' | 'z';
+
+export type ChannelName =
   | ChannelNameHSL
-  | ChannelNameXYZ
+  | ChannelNameHWB
+  | ChannelNameLAB
   | ChannelNameLCH
-  | ChannelNameLAB;
+  | ChannelNameRGB
+  | ChannelNameXYZ;
 
-type KnownColorSpace =
-  | ColorSpaceRGB
-  | ColorSpaceHWB
+export type KnownColorSpace =
   | ColorSpaceHSL
-  | ColorSpaceXYZ
-  | ColorSpaceLCH
-  | ColorSpaceLAB;
-
-type PolarColorSpace =
   | ColorSpaceHWB
-  | ColorSpaceHSL
+  | ColorSpaceLAB
   | ColorSpaceLCH
-  | ColorSpaceLAB;
+  | ColorSpaceRGB
+  | ColorSpaceXYZ;
 
-type RectangularColorSpace = Omit<KnownColorSpace, PolarColorSpace>;
+export type PolarColorSpace = ColorSpaceHSL | ColorSpaceHWB | ColorSpaceLCH;
 
-type HueInterpolationMethod =
-  | 'shorter'
-  | 'longer'
+export type RectangularColorSpace = Omit<KnownColorSpace, PolarColorSpace>;
+
+export type HueInterpolationMethod =
   | 'decreasing'
-  | 'increasing';
+  | 'increasing'
+  | 'longer'
+  | 'shorter';
 
-type ChannelValue = string | number | null;
+export type ChannelValue = number | null;
```

Comment:
I'm not sure this is worth defining as its own type rather than just declaring inline.